server/pages/index.js

```diff
@@ -119,7 +122,19 @@ function getCurrentCommitShortChecksum() {
 	}
 }
 
-function getDefaultContext( request ) {
+function getFileContentsSafely( filepath ) {
+	return new Promise( function( resolve ) {
+		fs.readFile( filepath, function( err, content ) {
```

Comment:
@ehg @DanReyLop I think caching in local vars would make a lot of sense. Both of those files _should_ be generated at build time, and then not change while the server is running. I'll make that change.

@ehg As for Promises, part of my reasoning was so I could use `Promise.all` [here](https://github.com/Automattic/wp-calypso/pull/15464/files#diff-b2d6784517d1151d7e75f0287d0c6478R201) and avoid waiting for one file read to finish before starting the next one. I could do that with callbacks, but Promises make it so much simpler! 😄 

In any case, caching the files in variables will mean that we will only actually have to read them once. So if we use sequential callbacks instead of Promises, it won't be much of an issue!